src/Gridwich.SagaParticipants.Publication.MediaServicesV3/tests/MediaServicesV3PublicationServiceTests.cs

```diff
@@ -242,7 +242,7 @@ public async void MediaServicesV3ServiceCreateLocatorTestWithPolicyUpdate(Reques
 
             // Arrange
             string assetName = "myassetname";
-            var amsAccount = new MediaService(storageAccounts: new List<StorageAccount>() { new StorageAccount() { Id = MediaServicesV3PublicationTestData.DefaultStorageId } });
+            var amsAccount = new MediaService(location: "westus", storageAccounts: new List<StorageAccount>() { new StorageAccount() { Id = MediaServicesV3PublicationTestData.DefaultStorageId } });
```

Comment:
Why location is hard coded "westus"?

Reply:
The AMS v6 SDK requests a location parameters when creating a media services account. It is hard coded but has no impact as this is for testing and the resources is not created in real.